### PR TITLE
Typography Updates

### LINKED ISSUE #51: Typography Updates
Make typography updates based on designs. Work Sans for everything except Futura for buttons

### PR DESCRIPTION
**Pull request by sreya-kata**
- Made all headings work sans and adjusted sizes, weights, and case
- Got rid of Futura Bold and Heavy
- Added type components for "connect" links and headings for people names (on team page)
- See photo below for visual of H2, HName, and P
fonts

### CODE REVIEW

File: src/constants/typography.js [hunk 1]
```diff
@@ -33,64 +34,51 @@ const textStyles = {
...
+  bold: `700`,
+  heavy: `900`,
 }
 
 const H1 = styled.h1`
   font-size: ${fontSizes.h1};
   font-family: ${fonts.header};
   font-weight: ${fontWeights.bold};
   text-transform: ${textStyles.uppercase};
+  letter-spacing: 1px;
```
Comment (Ryanjso) [inline src/constants/typography.js:58]: 👍🏻

File: src/constants/typography.js [hunk 2]
```diff
@@ -33,64 +34,51 @@ const textStyles = {
   uppercase: "uppercase",
   lowercase: "lowercase",
   capitalize: "capitalize",
+  underline: "underline",
```
Comment (Ryanjso) [inline src/constants/typography.js:37]: I think in some of the spots we might want to use bottom padding with a border bottom for custom underlines

**Review by Ryanjso — APPROVED** [review]
Looks good sista. Made a couple comments but yeah all good in my eyes

File: src/constants/typography.js (hunk 2 above)
Comment (sreya-kata) [reply src/constants/typography.js:37]: Got it! I just added a const for that in the types and used it for the connect link. The screenshot below is bottom-padding 0 but we can always change the style
connect

**Pull request merged**